corehq/apps/userreports/alembic_diffs.py

```diff
@@ -4,19 +4,18 @@
 from alembic.migration import MigrationContext
 
 
-def get_migration_context(connection, table_names=None, include_object=None):
+def get_migration_context(connection, table_names=None):
     opts = {'compare_type': True}
-
-    if callable(include_object):
-        opts['include_object'] = include_object
-    else:
-        opts['include_symbol'] = partial(include_symbol, table_names)
-
+    if table_names:
+        opts['include_object'] = partial(include_object, table_names)
```

Comment:
I'm realizing this might be changing behavior in a way we don't want. Since the `include_object` parameter was never used, we _always_ executed the `else` statement `opts['include_symbol'] = partial(include_symbol, table_names)`, even if `table_names` was None. In the event that `table_names` was None, `argument of type 'NoneType' is not iterable` would be thrown.

The only place `get_migration_context` is called without specifying `table_names` is https://github.com/dimagi/commcare-hq/blob/97410667267b6cbe329503f59ee71751ad194f9c/corehq/apps/userreports/rebuild.py#L103.